generated/maintenance/microsoft.maintenance/2018-06-01-preview/types.md

```diff
@@ -5,7 +5,8 @@
 ### Properties
 * **apiVersion**: '2018-06-01-preview' (ReadOnly, DeployTimeConstant): The resource api version
 * **id**: string (ReadOnly, DeployTimeConstant): The resource id
-* **name**: 'default' (Required, DeployTimeConstant): The resource name
+* **name**: 'default' | string (Required, DeployTimeConstant): The resource name
+* **properties**: [ApplyUpdateProperties](#applyupdateproperties) (ReadOnly): Properties of the apply update
 * **type**: 'Microsoft.Maintenance/applyUpdates' (ReadOnly, DeployTimeConstant): The resource type
```

Comment:
This resource defines its PUT with a constant name (the path ends in `/default`) and uses a parameterized path for the GET method targeting the same resource type at the same scope. What's being generated here is not incorrect IME, but this is an interesting collateral effect.